nautobot/extras/filters/__init__.py

```diff
@@ -345,12 +345,12 @@ class Meta:
 class CustomFieldChoiceFilterSet(BaseFilterSet):
     q = SearchFilter(filter_predicates={"value": "icontains"})
     field_id = django_filters.ModelMultipleChoiceFilter(
-        field_name="field",
+        field_name="custom_field",
         queryset=CustomField.objects.all(),
         label="Field",
     )
     field = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Should these be combined into a NaturalKeyOrPKMultipleChoiceFilter named `custom_field`?

Reply:
There are more than 100 cases of this, so I'll open a separate issue to merge "django filters.ModelMultipleChoiceFilter" into "NaturalKeyOrPKMultipleChoiceFilter." On the develop branch, this has already been completed.

Reply:
We have open issues for making this change across the board in 2.0 already. I just asked here since you're having to change this specific filter already, might as well move it to its final state.